journal/journal.go

```diff
@@ -77,34 +77,24 @@ func Send(message string, priority Priority, vars map[string]string) error {
 		return journalError("could not connect to journald socket")
 	}
 
+	socketAddr, err := net.ResolveUnixAddr("unixgram", "/run/systemd/journal/socket")
+	if err != nil {
+		return journalError(err.Error())
+	}
 	data := new(bytes.Buffer)
 	appendVariable(data, "PRIORITY", strconv.Itoa(int(priority)))
 	appendVariable(data, "MESSAGE", message)
 	for k, v := range vars {
 		appendVariable(data, k, v)
 	}
 
-	_, err := io.Copy(conn, data)
-	if err != nil && isSocketSpaceError(err) {
-		file, err := tempFd()
-		if err != nil {
-			return journalError(err.Error())
-		}
-		defer file.Close()
-		_, err = io.Copy(file, data)
-		if err != nil {
-			return journalError(err.Error())
-		}
-
-		rights := syscall.UnixRights(int(file.Fd()))
-
-		/* this connection should always be a UnixConn, but better safe than sorry */
-		unixConn, ok := conn.(*net.UnixConn)
-		if !ok {
-			return journalError("can't send file through non-Unix connection")
-		}
-		unixConn.WriteMsgUnix([]byte{}, rights, nil)
-	} else if err != nil {
+	/* this connection should always be a UnixConn, but better safe than sorry */
+	unixConn, ok := conn.(*net.UnixConn)
+	if !ok {
+		return journalError("can't send file through non-Unix connection")
+	}
+	_, _, err = unixConn.WriteMsgUnix(data.Bytes(), nil, socketAddr)
```

Comment:
For the reference, I found an old thread:
https://groups.google.com/forum/#!msg/golang-nuts/PyOeimgyvFw/VHNCxxEbDP0J

I think I'll try to reintroduce ce behaviour in the MR and see how we can fix it. Maybe it should be fixed directly in _journald-server_ (any contributor here ?)